docs/product-migration-guide/src/main/asciidoc/migration-execution-server-prep-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='migration-execution-server-prep-proc']
+= Prepare {CENTRAL} for migration
+
+Before you can migrate decision service data from {PRODUCT_OLD} 6.4, you must add to the new {CENTRAL} a server template and KIE containers corresponding to {PRODUCT_OLD} 6.4 process deployments.
+
+.Procedure
+. Start {PRODUCT_OLD} according to your installation (example, `$JBOSS_HOME/bin/standalone.sh`) and log in to {CENTRAL}.
+. Navigate to *Deploy -> Process Deployments* and note the *Deployment* identifier and other details for your deployments. This information will be required when you prepare the new {CENTRAL} in {PRODUCT} {PRODUCT_VERSION} for migration.
+. Close {CENTRAL} and stop {PRODUCT_OLD} 6.4 according to your installation.
+. Start {PRODUCT} {PRODUCT_VERSION} according to your installation (example,
```

Comment:
Not to harp on, but seems a little informal. I prefer (for example, ...)

Reply:
Sure. Was just trying to streamline, but you're right.